python/ray/data/tests/test_raydp_dataset.py

```diff
@@ -4,6 +4,7 @@
 import torch
 
 
+# RayDP tests require Ray Java. Make sure ray jar is built before running this test.
```

Comment:
Point to ci/env/install-java.sh so users know what exactly need to be installed?